packages/openapi-code-generator/src/typescript/client/typescript-axios/typescript-axios-client-builder.ts

```diff
@@ -22,7 +38,7 @@ export class TypescriptAxiosClientBuilder extends AbstractClientBuilder {
     const operationParameter = builder.methodParameter()
 
     const queryString = builder.queryString()
-    const headers = builder.headers()
+    const headers = builder.headers({nullContentTypeValue: "false"})
```

Comment:
Turns out that axios will always send a `Content-Type` header with `POST` requests, even if there isn't a body. This then ends up being parsed by the receiving server erroneously.

Setting the `Content-Type` header to `false` is the only way I've found to drop this behavior :cry: (it then omits the header completely, setting to `null` or `undefined` don't work)

https://axios-http.com/docs/config_defaults